common/SC_Apple.mm

```diff
@@ -41,5 +42,65 @@ void disableAppNap() {
     }
 }
 
+namespace EventLoop {
+
+std::atomic_bool bRunning;
```

Comment:
could you change this to `static std::atomic_bool g_running`? (in fact i would prefer `g_isRunning` but it's up to you). reasoning:

- prefer static for variables like these (only used in one TU) to indicate intent. also slightly decreases link time but i've never checked how much of a difference it makes with the tons of boost headers we use
- we generally don't use hungarian notation in this codebase, but we do use certain prefixes to indicate whether a variable is a data member, static data member, or global variable.

the C++ style guide used to not note anything about using `g_` as a prefix for globals, but i've added it now: https://github.com/supercollider/supercollider/wiki/Cpp-code-style-guidelines#naming

i think most global variables are currently named as `gSomeVar`, not `g_someVar`, but if we are advocating `m_` and `s_` we should also advocate `g_` for consistency. the underscore prefixes are used more in newer parts of the codebase, which (thankfully) don't use nearly as many global variables.

Reply:
done